crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -48,6 +48,7 @@ pub const HAS_RELATIVE_CONDITION: u32 = 2;
 // 3. No ASSERT_MY_COIN_ID condition, no more than one ASSERT_MY_PARENT_ID condition
 //    (as the second condition)
 // 4. it has an output coin with the same puzzle hash as the spend itself
+// 5. there are no ASSERT_MY_COIN_AMOUNT or AGGSIG_*_AMOUNT conditions
```

Comment:
good idea to amend this comment. you should add the other new restrictions too though. with sending/receiving messages and `ASSERT_BIRTH_*`, `ASSERT_EPHEMERAL` etc.

Reply:
changed this comment to reflect timelocks